src/experiment.py

```diff
@@ -0,0 +1,188 @@
+from optimisers import Problem, Optimiser
```

Comment:
All modules (i.e. `.py` files) should start with a docstring, just like classes, methods, and functions. (See PEP 257: https://peps.python.org/pep-0257/#specification). For all `.py` modules in cyclops it should be sufficient to include a one-line summary of what the file contains as well as a copyright declaration. Here's an example:

```suggestion
"""
Experiment class for cyclops.

(c) Copyright UKAEA 2023.
"""
from optimisers import Problem, Optimiser
```

(N.B. no problem with the import line, but on github I can only suggest editing line 1, not inserting new lines above it)